maintainer/format/autopep8.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Do we really need Bash? The original file was converted pure Dash ([`984b061`](https://github.com/espressomd/espresso/commit/984b061645a80f5e184eabf077a43ffddb0c26b8#diff-78719c369a7782258dc4e7743964d748)) to improve interoperability and prevent the re-introduction of bashisms.